hepdata/modules/email/utils.py

```diff
@@ -80,6 +81,13 @@ def send_email(destination, subject, message, reply_to_address=None):
         raise e
 
 
+def send_flask_message_email(message):
+    """
+    Creates a task to send an email from a flask_mail.Message instance
+    """
+    create_send_email_task(','.join(message.recipients), message.subject, message.html)
```

Comment:
Please add a final argument `reply_to_address=current_app.config['SECURITY_EMAIL_SENDER']` to send emails from `info@hepdata.net` rather than `submissions@hepdata.net`.

Reply:
> * Red flashed messages appear, for example, if a user tries to sign up with an existing email address or if the reCAPTCHA has not been completed.  I thought that these messages might be dismissed by clicking the red cross in the top-right, but it has no effect.  Does the red cross indicate a closable message (if so, something is broken) or does it just indicate an error?

Ah - it's another bit of JavaScript that I'd missed when I did the webpack rewrite. The dismiss button uses bootstrap JS, so I'll make sure it's added on every page.